packages/validator/test/unit/services/attestationDuties.test.ts

```diff
@@ -241,4 +242,137 @@ describe("AttestationDutiesService", () => {
 
     expect(api.validator.prepareBeaconCommitteeSubnet).toHaveBeenCalledOnce();
   });
+
+  describe("Reorg handling", () => {
+    const setupDuty = (slot: number): routes.validator.AttesterDuty => ({
+      slot,
+      committeeIndex: 1,
+      committeeLength: 120,
+      committeesAtSlot: 120,
+      validatorCommitteeIndex: 1,
+      validatorIndex: index,
+      pubkey: pubkeys[0],
+    });
+
+    async function setupTestEnvironment(duty: routes.validator.AttesterDuty, dependentRoot = "0xOLD_ROOT") {
```

Comment:
instead of these setup functions it would be better to make use of `beforeEach` hook to setup data, it's fine to be a bit repetitive (more verbose) in tests as it makes it easier to read them, encapsulations make it harder to understand what the test is doing